### ISSUE TITLE
Update Argparser

### ISSUE THREAD

**Issue by amillert** [amillert/pic2story#11]
Make sure all arguments needed for the module are present in the argparser and are being correctly interpreted in the main.py. Ensure proper logic and flow.

Some arguments:
- Batch size,
- Eta (learning rate),
- Dimensions of the nn,
- Amount of epochs,
- Loading pretrained weights, saving, etc.,
- Option to enable `tqdm` and `timer` logging.
